examples/resources/alkira_service_f5_lb/resource1.tf

```diff
@@ -13,6 +13,16 @@ resource "alkira_service_f5_lb" "example_lb_1" {
     f5_password     = "passwordispassword"
     f5_username     = "admin"
 
+  }
+  instances {
```

Comment:
I would rename `instances` to `instance`, since each block is 1 instance, but not instances...